presto-docs/src/main/sphinx/connector/mongodb.rst

```diff
@@ -146,7 +146,7 @@ This property is optional; the default is ``ACKNOWLEDGED``.
 ``mongodb.required-replica-set``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The required replica set name. With this option set, the MongoClient instance will
+The required replica set name. With this option set, the MongoClient instance::
 
 #. Connect in replica set mode, and discover all members of the set based on the given servers
```

Comment:
I think we need to adjust the phrasing of these given that we removed `will` above.

Reply:
Reworded